pyproject.toml

```diff
@@ -1,3 +1,50 @@
+[project]
+name = "PROJECT"
+authors = [
+    { name = "Jason R. Coombs", email = "jaraco@jaraco.com" },
+]
+description = "PROJECT_DESCRIPTION"
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+]
+requires-python = ">=3.8"
+dynamic = ["version"]
+dependencies = [
+]
+
+[project.optional-dependencies]
+testing = [
+    # upstream
+    "pytest >= 6, != 8.1.1",
+    "pytest-checkdocs >= 2.4",
+    "pytest-cov",
+    "pytest-mypy",
+    "pytest-enabler >= 2.2",
+    "pytest-ruff >= 0.2.1",
```

Comment:
Oh, this makes me sad. Now all dependencies will need to be wrapped in quotes and separated by commas. It's barely better than raw JSON. Oh, well. I guess that's the price to be paid for innovation, and certainly nothing you could have done about it.

Reply:
I checked if dependencies can maybe be stored in a long string, but checked the standard and it leaves no doubt there's nothing we could do about it: https://packaging.python.org/en/latest/specifications/pyproject-toml/#dependencies-optional-dependencies.